frontend/app/views/spree/checkout/_terms_and_conditions.en.html.erb

```diff
@@ -0,0 +1,64 @@
+____________________________________________________________
```

Comment:
I don't think `.html.erb` is appropriate here since html can't be rendered inside the textarea. Either this should be text, or the textarea should be changed to a different container.